resources/assets/enderio/lang/ru_RU.lang

```diff
@@ -332,7 +330,7 @@ tile.blockCapBank.activated.name=Блок конденсаторов
 tile.blockCapBank.vibrant.name=Вибрирующий блок конденсаторов
 tile.blockCapBank.tooltip.detailed.line1=Мульти-блок для хранения энергии.
 tile.blockCapBank.tooltip.detailed.line2=Может заряжать RF-предметы.
-tile.blockCapBank.tooltip.detailed.line3=Shift-ПКМ, по экрану пустой рукой,
+tile.blockCapBank.tooltip.detailed.line3=SHIFT+ПКМ, по экрану пустой рукой,
```

Comment:
"Shift" is a name of the key. Not an abbreviation.
Applies to the rest of occurences of this same name.